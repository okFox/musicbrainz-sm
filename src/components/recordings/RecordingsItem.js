import React from 'react';
import PropTypes from 'prop-types';
import { Link } from 'react-router-dom';



const RecordingsItem = ({ recordingTitle, artistName }) => {
  return (
    <Link to={`/lyrics/${artistName}/${recordingTitle}`}>
      <p>{recordingTitle}</p>
    </Link>
  );
};

RecordingsItem.propTypes = {
  recordingTitle: PropTypes.string.isRequired,
  artistName: PropTypes.string.isRequired
};

export default RecordingsItem;
