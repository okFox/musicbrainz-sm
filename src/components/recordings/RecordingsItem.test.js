import React from 'react';
import { shallow } from 'enzyme';
import RecordingsItem from './RecordingsItem';

describe('RecordingsItem component', () => {
  it('renders RecordingsItem', () => {
    const wrapper = shallow(<RecordingsItem recordingTitle="titel" artistName="name" />);
    expect(wrapper).toMatchSnapshot();
  });
});
