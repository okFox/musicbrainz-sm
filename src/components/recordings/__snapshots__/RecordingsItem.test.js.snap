// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`RecordingsItem component renders RecordingsItem 1`] = `
<Link
  to="/lyrics/name/titel"
>
  <p>
    titel
  </p>
</Link>
`;
